src/__tests__/incidents/report/ReportIncident.test.tsx

```diff
@@ -91,38 +89,38 @@ describe('Report Incident', () => {
     expect(categoryItemInput).toHaveDisplayValue('Warp Coil')
   })
 
-  test('renders a description formField element that allows user input', async () => {
+  it('renders a description formField element that allows user input', async () => {
     setup([Permissions.ViewIncident, Permissions.ResolveIncident])
-    const inputArr = await screen.findAllByRole('textbox', { name: /required/i })
-    const descriptionInput = inputArr[inputArr.length - 1]
+    const descriptionInput = screen.getByLabelText(/incidents\.reports\.description/i)
 
     expect(descriptionInput).toBeInTheDocument()
     expect(descriptionInput).toBeEnabled()
 
     userEvent.type(descriptionInput, 'Geordi requested analysis')
     expect(descriptionInput).toHaveDisplayValue('Geordi requested analysis')
   })
-  test(' renders action save button after all the input fields are filled out', async () => {
-    setup([Permissions.ViewIncident, Permissions.ResolveIncident])
-    const departmentInput = await screen.findByPlaceholderText(/incidents\.reports\.department/i)
-    const categoryInput = await screen.findByPlaceholderText(/incidents\.reports\.category\b/i)
-    const categoryItemInput = await screen.findByPlaceholderText(
-      /incidents\.reports\.categoryitem/i,
-    )
-    const inputArr = await screen.findAllByRole('textbox', { name: /required/i })
-    const descriptionInput = inputArr[inputArr.length - 1]
 
-    userEvent.type(departmentInput, 'Engineering Bay')
-    userEvent.type(categoryInput, 'Warp Engine')
-    userEvent.type(categoryItemInput, 'Warp Coil')
-    userEvent.type(descriptionInput, 'Geordi requested analysis')
+  // ! Remove test? Save button is always rendered regardless of input values
+  // it(' renders action save button after all the input fields are filled out', async () => {
```

Comment:
Is there already an integration test covering this, or do we need to create it?

Reply:
I'm not sure if there's an integration test for it already, but the thing it's trying to test doesn't happen in the staging app either; the "report incident" button is always visible.